docs/guides/dev_guide.rst

```diff
@@ -84,20 +84,37 @@ We also do a few things that aren't explicitly called out in PEP 8:
   width of 100 characters. As a result, our maximum line length is 100 characters,
   rather than 80.
 
+* Use parentheses ``()`` for line continuation::
+
+    # in imports
+    import (module1, module2, module3, module4,
+        module5)
+
+    # for long strings
```

Comment:
$.02: Agree with docstrings for multi-line, `()` for single line, discourage use of '\n' in strings.